pkg/backend/openshiftcluster.go

```diff
@@ -55,9 +57,15 @@ func (ocb *openShiftClusterBackend) try(ctx context.Context) (bool, error) {
 			ocb.m.EmitGauge("backend.openshiftcluster.workers.count", int64(atomic.LoadInt32(&ocb.workers)), nil)
 			ocb.cond.Signal()
 
-			ocb.m.EmitGauge("backend.openshiftcluster.duration", time.Now().Sub(t).Milliseconds(), map[string]string{
-				"state": string(doc.OpenShiftCluster.Properties.ProvisioningState),
-			})
+			// Emit duration metric for how long a create or delete takes to complete
+			if dequeuedState == api.ProvisioningStateCreating || dequeuedState == api.ProvisioningStateDeleting {
+				if ps := doc.OpenShiftCluster.Properties.ProvisioningState; ps == api.ProvisioningStateSucceeded || ps == api.ProvisioningStateFailed {
+					duration := ocb.now().Sub(doc.CorrelationData.RequestTime).Seconds()
+					ocb.m.EmitGauge("backend.openshiftcluster.duration", int64(duration), map[string]string{
+						"state": string(ps),
```

Comment:
Am I missing something but how do we know this metric was for Delete or Create? We set the state of the end result, but not "what was the intended action". And I somehow think this should 2 separate metrics for created and deletes as this is 2 separate intentions and code paths.

Reply:
@jim-minter do you agree with mj that we should emit durations with the action in them, e.g.:

```
backend.openshiftcluster.<action>.duration
```